packages/blocks/src/database-block/table/selection-manager/utils.ts

```diff
@@ -57,6 +57,11 @@ export function getClosestDatabaseId(element: Element) {
   return databaseId;
 }
 
+export function isInDatabase(element: Element) {
+  const database = element.closest('affine-database');
+  return database !== null;
+}
+
```

Comment:
Is it duplicated of `isInsideBlockByFlavour`?

Reply:
Their signatures are different, `isInDatabase` is used to determine whether the current element is in the Database.